libs/libc/misc/lib_mutex.c

```diff
@@ -177,7 +122,7 @@ int nxmutex_destroy(FAR mutex_t *mutex)
 
 bool nxmutex_is_hold(FAR mutex_t *mutex)
```

Comment:
why not inline nxmutex_is_hold

Reply:
_SCHED_GETTID requires including nuttx/sched.h, and including that in mutex.h causes some havoc. It's fine to have it in C file for now I suppose.